lib/node_modules/@stdlib/math/base/special/lcmf/README.md

```diff
@@ -0,0 +1,226 @@
+
+The [least common multiple][lcm] (lcm) of two non-zero integers `a` and `b` is the smallest positive integer that is divisible by both `a` and `b`. The lcm is also known as the **lowest common multiple** or **smallest common multiple** and finds common use in calculating the **lowest common denominator** (lcd).
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var lcmf = require( '@stdlib/math/base/special/lcmf' );
+```
+
+### lcmf(a,b)
+Computes the [least common multiple][lcm] for two single-precision floating-point numbers `a` and `b`.
+
+```javascript
+var v = lcmf( 10.0,12.0 );
+// returns 60
+```
+
+If either number is less than 1, the function returns `0`.
+
+```javascript
+var v = lcmf( 0.0, 12.0 );
+// returns 0
+
+v = lcmf( -5.0, 12.0 );
+// returns 0
+```
+
+The inputs must be valid floating-point numbers; otherwise, the function returns `NaN`.
+
+```javascript
+var v = lcmf( 3.14, 12.0 );
+// returns NaN
+
+v = lcmf( NaN, 12.0 );
+// returns NaN
+```
+
+</section> 
+
+<!-- /.usage --> 
+
+<!-- Package usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. --> 
+
+<section class="notes"> 
+
+</section> 
+
+<!-- /.notes --> 
+
+<!-- Package usage examples. --> 
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var lcmf = require( '@stdlib/math/base/special/lcmf' );
+
+var v;
+var i;
+
+for ( i = 1; i <= 10; i++ ) {
+    v = lcmf( i, 12 );
+    console.log( 'lcm(%d, 12) = %d', i, v );
+}
+```
+
+</section>
+
+<!-- /.examples -->
+
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/math/base/special/lcmf.h"
+```
+
+### stdlib_base_lcmf( a,b )
+
+Computes the [least common multiple][lcm] for  two floating-point numbers `a` and `b`.
+
+```c
+float v = stdlib_base_lcmf( 10.0, 12.0 );
+// returns 60.0
+```
+
+The function accepts the following argument:
+
+-   **a**: `[in] float` input value.
+-   **b**: `[in] float` input value.
+
+```c
+double stdlib_base_lcmf( const float a, const float b );
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- C API usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+</section>
+
+<!-- /.notes -->
+
+<!-- C API usage examples. -->
+
+<section class="examples">
+
+### Examples
+
+```c
+#include "stdlib/math/base/special/lcmf.h"
+#include <stdio.h>
+
+int main( void ) {
+    double v;
+    v = stdlib_base_lcmf( 10.0, 12.0 );
+    printf( "lcm(10.0, 12.0) = %lf\n", v );
+}
```

Comment:
Here too, we must include multiple examples. You can follow what we have in the double-precision implementation: https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/lcm/README.md?plain=1#L188